docs/usage/options.md

```diff
@@ -27,7 +27,7 @@ $ babel --name=value
 | `optional`               | `[]`                 | Array of transformers to [optionally](/docs/usage/transformers#optional) use. Run `babel --help` to see a full list of transformers. Optional transformers displayed inside square brackets. |
 | `nonStandard`            | `true`               | Enable support for JSX and Flow. |
 | `highlightCode`          | `true`               | ANSI highlight syntax error code frames |
-| `only`                   | `null`               | An array of [glob](https://github.com/isaacs/minimatch) paths to **only** compile. When attempting to compile a non-matching file it's returned verbatim. |
+| `only`                   | `null`               | A [glob](https://github.com/isaacs/minimatch) / regex or mixed array of both, matching pathes to **only** compile. Can also be an array of arrays containing pathes to explicitly match. When attempting to compile a non-matching file it's returned verbatim. |
```

Comment:
Still not sure how I got it managed to write `pathes` all the time.
Thanks for the little language enhancement :+1: